qiskit_ibm_runtime/options/options.py

```diff
@@ -155,6 +169,56 @@ def _get_program_inputs(options: dict) -> dict:
                 inputs[key] = options[key]
         return inputs
 
+    @staticmethod
+    def _validate_program_inputs(
```

Comment:
```suggestion
    def _validate_options(
```
nit - "options" is what users specify, and "program inputs" is what go to the runtime programs. They are different to keep backward compatibility. This function is validating the options, not the program inputs.